packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -585,13 +855,64 @@ class RoundSliderThumbShape extends SliderComponentShape {
       end: sliderTheme.thumbColor,
     );
     canvas.drawCircle(
-      thumbCenter,
+      center,
       radiusTween.evaluate(enableAnimation),
       Paint()..color = colorTween.evaluate(enableAnimation),
     );
   }
 }
 
+/// This is the default shape to a [Slider]'s thumb overlay if no
+/// other shape is specified.
+///
+/// See also:
+///
+///  * [Slider] for the component that this is meant to display this shape.
+///  * [SliderThemeData] where an instance of this class is set to inform the
```

Comment:
[SliderTheme], which can be used to configure the the thumb shape of all sliders in a widget subtree.